src/Layer/TiledGeometryLayer.js

```diff
@@ -1,10 +1,16 @@
+import * as THREE from 'three';
 import GeometryLayer from 'Layer/GeometryLayer';
 import { InfoTiledGeometryLayer } from 'Layer/InfoLayer';
 import Picking from 'Core/Picking';
 import convertToTile from 'Converter/convertToTile';
 import CancelledCommandException from 'Core/Scheduler/CancelledCommandException';
 import ObjectRemovalHelper from 'Process/ObjectRemovalHelper';
+import { SIZE_TEXTURE_TILE } from 'Provider/OGCWebServiceHelper';
 
+const subdivisionVector = new THREE.Vector3();
+const boundingSphereCenter = new THREE.Vector3();
+
+export const diagonalTilePx = Math.pow(2 * (SIZE_TEXTURE_TILE * SIZE_TEXTURE_TILE), 0.5);
```

Comment:
Could you add precision on this constant ? The px suffix is not explicit enough.